optaplanner-benchmark/src/test/java/org/optaplanner/benchmark/impl/io/jaxb/OffsetDateTimeJaxbAdapterTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.benchmark.impl.io.jaxb;
```

Comment:
Should this be in optaplanner-persistence-jaxb's package instead? Definitely impl

Reply:
I am not in favor of moving the adapter to the optaplanner-persistence-jaxb module.

The optaplanner-persistence-* modules care about OptaPlanner-specific types, which at the moment are only the Score types.